test/streamparse/test_bolt.py

```diff
@@ -0,0 +1,308 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import itertools
+import json
+import logging
+import signal
+import time
+import unittest
+from functools import partial
+from io import BytesIO
+
+try:
+    from unittest import mock
+    from unittest.mock import patch
+except ImportError:
+    import mock
+    from mock import patch
+
+from six.moves import range
+
+from streamparse.storm import BatchingBolt, Bolt, Component, Tuple
+
+
+log = logging.getLogger(__name__)
+
+
+class BoltTests(unittest.TestCase):
+
+    def setUp(self):
+        self.tup_dict = {'id': 14,
+                         'comp': 'some_spout',
+                         'stream': 'default',
+                         'task': 'some_bolt',
+                         'tuple': [1, 2, 3]}
+        tup_json = "{}\nend\n".format(json.dumps(self.tup_dict))
+        self.tup = Tuple(self.tup_dict['id'], self.tup_dict['comp'],
+                         self.tup_dict['stream'], self.tup_dict['task'],
+                         self.tup_dict['tuple'],)
+        self.bolt = Bolt(input_stream=itertools.cycle(tup_json.splitlines(True)),
+                         output_stream=BytesIO())
```

Comment:
I don't use the `output_stream` bit in any tests yet, but if we wanted to, we could double-check that the write things end up in stdout by checking `self.bolt.output_stream.getvalue()`.